src/darwin/Framework/CHIP/templates/templates.json

```diff
@@ -132,6 +132,11 @@
             "path": "MTRClusterConstants.zapt",
             "name": "Constants for cluster IDs",
             "output": "src/darwin/Framework/CHIP/zap-generated/MTRClusterConstants.h"
+        },
+        {
+            "path": "MTRAttributeUtils-src.zapt",
+            "name": "Utility functions for attributes",
+            "output": "src/darwin/Framework/CHIP/zap-generated/MTRAttributeUtils.mm"
```

Comment:
Probably OK, but maybe we should just call this MTRSpecifiedAttributeCheck.h/mm so we don't start creating too-big "utils" files.

Reply:
Fixed in the last commit.